command/server/server.go

```diff
@@ -168,6 +168,16 @@ func setFlags(cmd *cobra.Command) {
 		),
 	)
 
+	cmd.Flags().Uint64Var(
+		&params.rawConfig.PubSubBuffersSize,
+		pubSubBufferSize,
+		defaultConfig.PubSubBuffersSize,
+		fmt.Sprintf(
+			"PubSub buffers size. Default: %ds",
```

Comment:
Why unit is `s`?

Reply:
This was only draft PR, nothing ready for review, will ping you once i prepare PR, thank you anyways for review!